### DISCUSSION TITLE
code space

### DISCUSSION THREAD

**Discussion by Mahdi-Re** [community/community#134480] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hi guys ! How to use github code?

**Comment by Mahd25** [comment, marked as answer]
**Hi!**

I will show you in some pictures:

#### 1:
![code-space01](https://github.com/user-attachments/assets/89c6116d-50e7-473e-9f04-5aea54c7b811)

#### 2:
![code-space02](https://github.com/user-attachments/assets/1965e8be-456a-4047-81a1-dca9c0be1c90)

#### 3:
![code-space03](https://github.com/user-attachments/assets/aeb17bf0-dbae-412d-be84-f89a810f7f62)


### And after the changes, you need to save them to be applied on GitHub:
![saved-changes](https://github.com/user-attachments/assets/aaa87017-a8aa-423b-903a-b46976a468f5)

**Reply by Mahd25** [reply]
I'm glad it was useful to you
